### DISCUSSION TITLE
Disable Maven scanning snapshots dependency

### DISCUSSION THREAD

**Discussion by patrickpannn** [renovatebot/renovate#24228] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I have been trying to force renovate to not scann the snapshot dependency using "allowedVersions" field as in below, but it seems not working. Can someone please have a look at it? Any help would be appreciated!

module.exports = {
    hostRules: [
      {
        hostType: 'maven',
        matchHost: '...'
      }
    ],
    packageRules: [
      {
        matchDatasources: ['maven'],
        allowedVersions: "!/SNAPSHOT/"
      }
    ]
}

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: HEAD .../org/springframework/boot/spring-boot-starter-parent/1.4.0.BUILD-SNAPSHOT/spring-boot-starter-parent-1.4.0.BUILD-SNAPSHOT.pom = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=21297) (repository=abc, baseBranch=abc)
DEBUG: HEAD .../org/springframework/boot/spring-boot-starter-parent/1.4.1.BUILD-SNAPSHOT/spring-boot-starter-parent-1.4.1.BUILD-SNAPSHOT.pom = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=20823) (repository=abc, baseBranch=abc)
DEBUG: HEAD .../org/springframework/boot/spring-boot-starter-parent/1.4.2.BUILD-SNAPSHOT/spring-boot-starter-parent-1.4.2.BUILD-SNAPSHOT.pom = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=17402) (repository=abc, baseBranch=abc)```

</details>


### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
Duplicate of #24227

**Reply by patrickpannn** [reply]
have closed that one

**Reply by rarkins** [reply]
I've answered in the other because this is the one marked as duplicate
